index.js

```diff
@@ -1,24 +1,140 @@
-        <hr>
-    </div>
+    <tr class="tableRow">
+        <td>"${book.title}" by ${book.author} </td>   
+        <td><button id ="delete${index}">Remove</button></td>
+    </tr>
     `;
   return content;
 }
 
-function storeForm(form) {s
-  localStorage.setItem('form', JSON.stringify(form));
+const addButton = document.getElementById('addBook');
+const newTitle = document.getElementById('bookTitle');
+const newAuthor = document.getElementById('authorName');
+const container = document.querySelector('.bodyh1');
+const navBar = document.querySelector('.nav-bar');
+
+for (let i = 0; i < books.length; i += 1) {
+  const book = displayBooks(books[i], i);
+  container.innerHTML += book;
 }
-s
\ No newline at end of file
+
+addButton.addEventListener('click', (e) => {
+  e.preventDefault();
+  if (newAuthor.value !== '' && newTitle.value !== '') {
+    const library = new Books(newAuthor.value, newTitle.value, books);
+    library.addBook();
+    storeForm(books);
+    window.location.reload();
+  }
+  document.getElementById('addNewForm').reset();
+});
+
+const deleteBtn = document.querySelectorAll('[id^="delete"]');
+deleteBtn.forEach((button) => {
+  button.addEventListener('click', () => {
+    const libraryOne = new Books(newAuthor.value, newTitle.value, books, button);
+    libraryOne.deleteBook();
+    storeForm(books);
+  });
+});
+
+// SPA
+
+const navHome = document.querySelector('.nav-home');
+const navBooks = document.querySelector('.nav-books');
+const navContact = document.querySelector('.nav-contact');
+
+const homeSection = document.querySelector('.home');
+const bookSection = document.querySelector('.books');
+const contactSection = document.querySelector('.contacts');
+
+navHome.addEventListener('click', () => {
+  navHome.style.color = 'blue';
+  navBooks.style.color = 'black';
+  navContact.style.color = 'black';
+  homeSection.style.display = 'flex';
+  bookSection.style.display = 'none';
+  contactSection.style.display = 'none';
+});
+navBooks.addEventListener('click', () => {
+  navHome.style.color = 'black';
+  navBooks.style.color = 'blue';
+  navContact.style.color = 'black';
+  homeSection.style.display = 'none';
+  bookSection.style.display = 'flex';
+  contactSection.style.display = 'none';
+});
+const handleNavClick = (navItem, navItems, sections) => {
+  // Reset all nav items and sections
+  navItems.forEach((item) => {
+    item.style.color = 'black';
+  });
+  sections.forEach((section) => {
+    section.style.display = 'none';
+  });
+
+  // Highlight selected nav item and show corresponding section
+  navItem.style.color = 'blue';
+  const { sectionId } = navItem.dataset;
+  const section = document.getElementById(sectionId);
+  if (section) {
+    section.style.display = 'flex';
+  }
+};
+
+navContact.addEventListener('click', () => {
+  const navItems = navBar.querySelectorAll('.nav-item');
+  const sections = document.querySelectorAll('.section');
+  handleNavClick(navContact, navItems, sections);
+});
```

Comment:
- [ ] When the contact link is clicked on the navigation bar, nothing happens. Kindly make sure you reached correct elements in the dom and return the right section.

Reply:
fix it thank you 👏
 You can find this project here: [https://github.com/kayjeee/bookstore].  Please leave a star it helps. Thanks for checking out my code!